src/libraries/Microsoft.Extensions.Primitives/tests/Microsoft.Extensions.Primitives.Tests.csproj

```diff
@@ -4,6 +4,19 @@
     <TargetFrameworks>$(DefaultNetCoreTargetFramework);net472</TargetFrameworks>
```

Comment:
I think this needs to change to $(NetCoreAppCurrent).  Without it these tests aren't running.